framework/set/setRKE1.go

```diff
@@ -171,10 +149,9 @@ func SetRKE1(t *testing.T, k8sVersion string, nodePools []config.Nodepool, file
 
 	rootBody.AppendNewline()
 
-	_, err = file.Write(f.Bytes())
-
+	_, err = file.Write(newFile.Bytes())
 	if err != nil {
-		t.Logf("Failed to write RKE1 configurations to main.tf file. Error: %v", err)
+		logrus.Infof("Failed to write AKS configurations to main.tf file. Error: %v", err)
```

Comment:
copy/paste fluke -- but this should mention RKE1 and not AKS here :)

Reply:
Resolved